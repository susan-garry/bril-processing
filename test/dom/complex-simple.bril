#ARGS: frontier
@main {
.block0:
    cond1: bool = const false;
.for_cond:
    br cond1 .for_body .for_end;
.for_body:
    br cond1 .then .else;
.then: 
    jmp .endif;
.else:
    cond1: bool = not cond1;
.endif:
    jmp .for_cond;
.for_end:
}