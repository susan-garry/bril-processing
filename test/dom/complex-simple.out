[('block0', []), ('else', ['endif']), ('endif', ['for_cond']), ('for_body', ['for_cond']), ('for_cond', ['for_cond']), ('for_end', []), ('then', ['endif'])]
